### DISCUSSION TITLE
[BUG] ModuleNotFoundError in Unreal Engine Python

### DISCUSSION THREAD

**Discussion by ZQlby** [dashingsoft/pyarmor#1171] [General]
LogPython: Error: Traceback (most recent call last):
LogPython: Error:   File "D:/UE_5.1/Engine/Plugins/VirtualProduction/MP/Content/Python/test_functions/dist/test.py", line 4, in <module>
LogPython: Error:     from pyarmor_runtime_000000 import __pyarmor__
LogPython: Error: ModuleNotFoundError: No module named 'pyarmor_runtime_000000'
=========================================================
(run generated py file in unreal engine show this error)
work fine in powershell, but not in unreal engine 
is it because unreal engine embedded python not  support pyd file？
any solution aviable
big thx ahead

**Comment by jondy** [comment]
I know nothing about `Unreal Engine Python`, but if it could not load binary extension, the obfuscated scripts may not work.
